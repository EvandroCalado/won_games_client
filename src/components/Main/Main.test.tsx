import { render, screen } from '@testing-library/react';
import { Main } from '.';

describe('Main', () => {
  it('should render heading', () => {
    render(<Main />);
    expect(screen.getByRole('heading', { level: 1 })).toBeInTheDocument();
  });

  it('should render a snapshot', () => {
    const { container } = render(<Main />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
