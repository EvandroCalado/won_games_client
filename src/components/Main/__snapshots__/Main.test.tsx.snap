// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main should render a snapshot 1`] = `
<main
  class="flex min-h-screen flex-col items-center justify-center space-y-8 bg-zinc-100 p-24 dark:bg-zinc-900"
>
  <h1
    class="text-custom text-4xl font-bold"
  >
    Next Boilerplate Tailwind
  </h1>
  <img
    alt="Logo"
    data-nimg="1"
    decoding="async"
    height="40"
    loading="lazy"
    src="/logo.svg"
    style="color: transparent;"
    width="300"
  />
</main>
`;
